src/store/persona/slice.ts

```diff
@@ -0,0 +1,61 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { initialState } from 'store/persona/types';
+import { createPersonaUser, fetchPersonaUser } from 'store/persona/thunks';
+
+// Since api can return null for personaUserInfo, undefined must be set by default
+export const PersonaState: initialState = {
+  personaUserInfo: undefined,
+  isLoading: true,
+  isUpdating: false,
+  isDeleting: false,
+};
+
+const personaSlice = createSlice({
+  name: 'persona',
+  initialState: PersonaState,
+  reducers: {
+    cleanLogout: (state) => {
+      return PersonaState;
+    },
+    setIsPersonaLoading: (state, action: PayloadAction<boolean>) => ({
+      ...state,
+      isLoading: action.payload,
+    }),
+  },
+  extraReducers: (builder) => {
+    // Create Persona User
+    builder.addCase(createPersonaUser.pending, (state) => ({
+      ...state,
+      isLoading: true,
+      error: undefined,
```

Comment:
I think that null would be more appropriate (valid for all other errors)

Reply:
`undefined' for error is already used on all other store, change it mean to update all the stores